src/controllers/pallets/PalletsStorageController.ts

```diff
@@ -48,20 +52,24 @@ export default class PalletsStorageController extends AbstractController<Pallets
 	protected initRoutes(): void {
 		// TODO look into middleware validation of in path IDs. https://github.com/paritytech/substrate-api-sidecar/issues/281
 		this.safeMountAsyncGetHandlers([
-			['/:storageItemId', this.getStorageItem],
+			['/:storageItemId', this.getStorageItem as RequestHandler],
 			['/', this.getStorage],
 		]);
 	}
 
-	private getStorageItem: RequestHandler = async (
+	private getStorageItem: RequestHandler<
+		IPalletsStorageParam,
+		unknown,
```

Comment:
What are these unknown values here?

Reply:
The `unknown` values are keys we dont need to specify in the RequestHandler. 

For example heres the RequestHandler interface type

```
RequestHandler<P = core.ParamsDictionary, ResBody = any, ReqBody = any, ReqQuery = qs.ParsedQs, Locals extends Record<string, any> = Record<string, any>>
```

So here you can see we dont need to specify any time for ResBody, or ReqBody so we set those to `unknown`

Reply:
Got it TIL🎉